### ISSUE TITLE
[REC] VDH Consuming Fire never recommended (wrong name in action list)

### ISSUE THREAD

**Issue by dubudevs** [Hekili/hekili#3953]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Spec

DH - Vengeance

### Describe the Issue

Forgot I changed this one, Immolation Aura is changed to Consuming Fire in meta (only meta meta not fel dev meta) so the name needs to be changed on action 5 of the metamorphosis list.

### How to Reproduce

.

### Snapshot (Link)

.

**Comment by taherbert** [comment]
In SimC, something like `action+=/immolation_aura` will be automatically converted to consuming flame while inside of meta. Consuming Flame isn't implemented as a distinct ability. When you see non-empowered abilities in the APL inside of the meta/fel dev action lists, it's because those empowered abilities aren't implemented in SimC. In-game, all of the empowered abilities have their own spell IDs.

**Comment by dubudevs** [comment]
> In SimC, something like `action+=/immolation_aura` will be automatically converted to consuming flame while inside of meta. Consuming Flame isn't implemented as a distinct ability. When you see non-empowered abilities in the APL inside of the meta/fel dev action lists, it's because those empowered abilities aren't implemented in SimC. In-game, all of the empowered abilities have their own spell IDs.

Most spells are correctly changed, but this is an exception.

**Comment by Hekili** [comment]
I need a snapshot of Consuming Fire not being recommended in Meta to determine a resolution. The intent is that Immolation Aura functions for both, as noted above.

**Comment by dubudevs** [comment]
https://github.com/Hekili/hekili/issues/3940 needs to be resolved first, which spells are granting which other spells and which cds are being reset is all over the place and not accurate to in game at all, so I do not know whether its because consuming flame is marked incorrectly or the addon just doesnt know whats going on.

**Comment by Hekili** [comment]
> https://github.com/Hekili/hekili/issues/3940 needs to be resolved first, which spells are granting which other spells and which cds are being reset is all over the place and not accurate to in game at all, so I do not know whether its because consuming flame is marked incorrectly or the addon just doesnt know whats going on.

All issues are going to get resolved, but FWIW, the issues with CD resets being off only impacts *future* recommendations.  When you use Metamorphosis or Fel Devastation or any ability, the addon updates to reflect what was actually changed.  So providing the snapshot would still help and there's no reason to wait.

Waiting just means that this particular issue is delayed, since you'll have to wait for the #3940 fix to be released and then update this ticket and its resolution will wait until the next release.

tl;dr - Just grab the snapshot, bro.

**Comment by dubudevs** [comment]
Didnt see your reply here, my bad :) Here's a snapshot during meta with Consuming Fire ready using the updated VDH profile with the current default APL. 

https://pastebin.com/X7jDWVqP

In the meta action list everything uses its meta empowered ability names (sigil of doom, fel devestation), except consuming fire which is listed as immo aura. https://i.imgur.com/ewIC2K3.png

**Comment by dubudevs** [comment]
I noticed that the meta list is not consistently being called while in meta, causing the meta empowered spells to not be recommended ~~after demonsurge has been consumed (?)~~ due to incorrectly calculating which version of meta you are in.

**Comment by Hekili** [comment]
This is resolved with the #3940 fix.
